### PR TITLE
adding commands summary script for quick reference

### PR DESCRIPTION
**Pull request by wjallen**
https://asciinema.org/a/KwOdCnqb6ogDofNppHA5hPnjq

### CODE REVIEW

File: bin/subcommands/commands [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+  monitors          Print monitors command summaries
+  notifications     Print notifications command summaries
+  postits           Print postits command summaries
+  profiles          Print profiles command summaries
+  requestbin        Print requestbin command summaries
+  systems           Print systems command summaries
+  tags              Print tags command summaries
+  tenants           Print tenants command summaries
+  transforms        Print transforms command summaries
+  abaco             Pring abaco command summaries
```
Comment (mwvaughn) [inline bin/subcommands/commands]: Print abaco command summaries

**Review by mwvaughn — CHANGES_REQUESTED** [review]
I approve, pending a fix to the "pring" typo in the abaco summary.

**Comment by wjallen** [comment]
I fixed the typo

**Review by johnfonner — APPROVED** [review]
Could replace the long "elif" chain with a case statement, but it works.  Nice functionality addition.

**Pull request merged**